src/main/java/net/pms/util/CoverArtArchiveUtil.java

```diff
@@ -493,7 +493,7 @@ private static void releaseCoverLatch(CoverArtArchiveCoverLatch coverLatch) {
 				connection = MediaDatabase.getConnectionIfAvailable();
 				// Check if it's cached first
 				if (connection != null) {
-					CoverArtArchiveResult result = MediaTableCoverArtArchive.findMBID(connection, mBID);
+					CoverArtArchiveResult result = MediaTableCoverArtArchive.findMBID(mBID);
```

Comment:
Why is the connection sharing removed throughout this file?

Reply:
Because sharing wasn't used ... the connection pool is sufficient unless you'd like to batch process something ...